.travis.yml

```diff
@@ -22,7 +22,7 @@ services:
 
 env:
   global:
-  - CORE_BRANCH=master
+  - CORE_BRANCH=stable12
```

Comment:
That's what #181 will do. Basically I want this app to be exclusively compatible to nc12 for v1.3.1 and then do the jump to nc13.